src/main/java/seedu/contax/model/appointment/Priority.java

```diff
@@ -0,0 +1,12 @@
+package seedu.contax.model.appointment;
+
+/**
+ * Represents an {@link Appointment}'s priority
+ **/
+
+public enum Priority {
+    LOW,
+    MEDIUM,
+    HIGH,
+    NONE
```

Comment:
What is the purpose of `NONE` if you are doing a nullable priority?

Reply:
this is used in model equals checking to get like a dummy representation for null, or else null will throw execption